client/Lykke.Service.PayInternal.Client/Models/Transfer/TransferRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Lykke.Service.PayInternal.Client.Models.Transfer
+{
+    public class TransferRequest 
+    {
+        public string TransferId { get; set; }
```

Comment:
Why do we need to fill in transferId? It is a request and supposed to generate id internally.